copulas/univariate/truncnorm.py

```diff
@@ -19,12 +19,18 @@ class TruncNorm(ScipyWrapper):
 
     def fit(self, X):
         """Prepare necessary params and call super().fit."""
-        min_ = X.min() - EPSILON
-        max_ = X.max() + EPSILON
-        self.mean = X.mean()
-        self.std = X.std()
+        self.constant_value = self._get_constant_value(X)
+        if self.constant_value is None:
+            min_ = X.min() - EPSILON
+            max_ = X.max() + EPSILON
+            mean = X.mean()
+            std = X.std()
 
-        super().fit(X, min_, max_)
+            super().fit(X, min_, max_, loc=mean, scale=std)
```

Comment:
Let's not use `loc` and `scale` here to keep this a truly Normal distribution and rather create an issue to add a TruncGaussian distribution that uses them.